### ISSUE TITLE
Formatting

### ISSUE THREAD

**Issue by thecodeduchess** [ct315/prj-rev-bwfs-tea-cozy#2]
Does it look this spaced out in your text editor? 

<img width="850" alt="screen shot 2018-06-08 at 2 17 45 pm" src="https://user-images.githubusercontent.com/8722966/41176898-b794db1a-6b27-11e8-9562-1f841bef3340.png">

**Comment by ct315** [comment]
No, that looks really weird. My eyes would go crazy looking at that!
